ebpf/address.go

```diff
@@ -0,0 +1,95 @@
+package ebpf
+
+import (
+	"encoding/binary"
+	"net"
+
+	"github.com/mailru/easyjson/jwriter"
+)
+
+// Address is an IP abstraction that is family (v4/v6) agnostic
+type Address interface {
+	Bytes() []byte
+	String() string
+	MarshalEasyJSON(w *jwriter.Writer)
```

Comment:
nit: I believe you can embed the `easyjson.Marshaler` interface here for the same effect

Reply:
I purposely left it out to not include an `Unmarshal` function. I'm hoping we'll be able to rip all this out when we change to protobuf for process-agent <> network-tracer :)

Reply:
Understood.  FWIW, the `Marshaler` interface only contains the `MarshalXXX` method